newrelic-agent/src/test/java/com/newrelic/agent/attributes/AttributeValidatorTest.java

```diff
@@ -186,7 +187,24 @@ public void testVerifyTruncatedValue() {
         assertEquals(expected, result);
     }
 
-    // TODO testVerifyTruncatedValue for LogEvent data
+    @Test
+    public void testVerifyTruncatedValueForLogEventData() {
+        Map<String, Object> input = new HashMap<>();
+        String longValue = Strings.padEnd("", 33000, 'e');
+        String longExpectedValue = Strings.padEnd("", 32767, 'e');
```

Comment:
Do you want these hard coded like other test or use `ConfigConstant`?

Reply:
The convention I've seen is that if they aren't used over again, then this is what folks have done in the past.  If the tests grow, I think someone will move them to constants.  That's my junior 2cents worth.